.eslintignore

```diff
@@ -1,7 +1,8 @@
-node_modules
+**/node_modules
 flow-typed/**/*.js
 coverage
 **/dist
+**/build
```

Comment:
Temporary... the math-input project had a `build/` folder.  This sped up ESLint a ton to skip processing these build files.

Reply:
Sneaky, but won't this prevent .js files in config/build from being processed?

Reply:
Nice catch!  Fixed.